CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
-## [4.39.0](https://github.com/voxpupuli/beaker/tree/4.39.0) (2023-02-09)
+## [4.40.0](https://github.com/voxpupuli/beaker/tree/4.40.0) (2023-04-04)
+
+[Full Changelog](https://github.com/voxpupuli/beaker/compare/4.39.0...4.40.0)
+
+**Fixed bugs:**
```

Comment:
If this is a bugfix, then why is this a minor version bump? Shouldn't it be considered an enhancement?

Reply:
Yes good point, I updated it to an enhancement